src/Core/Resolvers/Sql Query Structures/BaseSqlQueryStructure.cs

```diff
@@ -346,8 +346,20 @@ protected List<LabelledColumn> GenerateOutputColumns()
         /// <param name="systemType"></param>
         /// <returns></returns>
         /// <exception cref="NotSupportedException"></exception>
-        protected static object ParseParamAsSystemType(string param, Type systemType)
+        protected object ParseParamAsSystemType(string param, Type systemType)
         {
+            if (systemType.Name.Equals("DateTime"))
+            {
+                if (ApiType is ApiType.GraphQL)
+                {
+                    // To continue supporting 'Z' notation for GraphQL, we parse datetime values as datetimeoffset.
```

Comment:
Would be helpful to add links to `DateTimeOffset` and `DateTime` docs in the comments.